APCAppCore/APCAppCore/Library/Objects/APCTasksReminderManager.m

```diff
@@ -672,4 +672,67 @@ -(BOOL)includeTaskInReminder:(APCTaskReminder *)taskReminder{
     return includeTask;
 }
 
+/*********************************************************************************/
+#pragma mark - Task Reminder Helper For Determining "new tasks"
+/*********************************************************************************/
+
++ (NSArray*)findNewTasksFromOld:(NSArray*)  oldTaskGroups
+                toNewTaskGroups:(NSArray*)  newTaskGroups
+   withActivityReminderDuration:(NSInteger) daysIncomplete
+{
+    NSArray* oldTasks = [oldTaskGroups valueForKey:@"task"];
+    NSMutableArray* oldTaskIds = [[oldTasks valueForKey:@"taskID"] mutableCopy];
+    
+    NSMutableDictionary* oldTaskCompletedStatusById = [@{} mutableCopy];
+    NSMutableDictionary* oldTaskById                = [@{} mutableCopy];
+    
+    for (APCTaskGroup* taskGroup in oldTaskGroups) {
+        oldTaskCompletedStatusById[taskGroup.task.taskID] = @(taskGroup.isFullyCompleted);
+        oldTaskById[taskGroup.task.taskID]                = taskGroup.task;
+    }
+    
+    // Right now there are a few conditions for determining if a task should be reminded about...
+    // 1) for any task, if the taskId was not in the old list, and is now in the new list
+    // 2) for any task, if the task was completed in old, but is incomplete in new
+    // 3) for activities, if the task is incomplete, old task is less than 3 days old, and new
+    //    task is more than 3 days old
+    NSMutableArray* newTasks = [@[] mutableCopy];
+    for (APCTaskGroup* taskGroup in newTaskGroups)
+    {
```

Comment:
My bad, I'm usually pretty good about that.  Each library has different conventions, so I forget which one I'm editing sometimes.  Will fix.